### ISSUE TITLE
npm WARN "deprecated gulp-minify-css" on semantic-ui 2.2.3

### ISSUE THREAD

**Issue by neokio** [Semantic-Org/Semantic-UI#4463]
When doing an npm update, semantic went from 2.2.2 to 2.2.3, and this error appeared:

```
UNMET PEER DEPENDENCY gulp-minify-css@*
npm WARN semantic-ui@2.2.3 requires a peer of gulp-minify-css@* but none was installed.
npm WARN deprecated gulp-minify-css@1.2.4: Please use gulp-clean-css
```

Solved with a simple `npm install gulp-minify-css --save-dev`, of course, but you know, when npm says an unmet peer is deprecated, I get a bit queasy and gassy ;)

**Comment by jlukic** [comment]
`minify-css` is no longer being used. We've updated to `clean-css` in `2.2.3` unfortunately forgot to wipe it from the `package.json` will remove in next bump.

**Comment by jlukic** [comment]
Fixed in `2.2.4`
